### DISCUSSION TITLE
fake custom pitch

### DISCUSSION THREAD

**Discussion by Ragdoll117** [Fedoraware/Fedoraware#1110] [Ideas]
Probably the 3rd fake pitch option, can be used to look more legit

**Comment by ghost** [comment]
Your fake pitch can either be up or down due to the way fake pitches work.

I believe when the server calculates your hitboxes, it uses the module operator to make sure that they are within the bounds of 89 and -89, as opposed to clamping them;
(720) % 90 = 0
std::clamp(720, -90, 90) = 90

for the above example, we are sending a pitch of 720, when the server calculates our hitbox it uses the module operator, meaning for the server our pitch is 0, when it sends it to other players however it clamps the value between -90 and 90, meaning for other players our pitch (in this example) would be 90 (down). Custom real pitches simply abuse the fact that the server uses different methods when calculating our pitch vs when it sends it to other players, but this also means that we cannot have a "custom fake" pitch, as for a fake pitch to occur, the pitch sent to other players will be bound to either 90 or -90.

**Reply by ghost** [reply]
I'm only explaining this because it seems like a good question if you don't understand what's happening, and I want to have something I can reference to tell people that it's not possible.
